runtime/compiler/x/codegen/J9CodeGenerator.cpp

```diff
@@ -126,6 +126,8 @@ J9::X86::CodeGenerator::initialize()
       cg->setSupportsStackAllocationOfArraylets();
       }
 
+   cg->setSupportsInlineVectorizedHashCode();
```

Comment:
Is this supported on both 32-bit and 64-bit?  If 64-bit only, this needs to be guarded.

Reply:
I'm trying to think if there are any technical reasons it can't run on 32-bit. It probably can but I haven't tested it.

Reply:
needs to add min target requirement of sse 4.1